Docker.DotNet.X509/PrivateKey.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace Docker.DotNet.X509
+{
+    public static class PrivateKey
```

Comment:
This can also be internal. We only need to expose the load functions from `CertificateCredential.cs`